jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -369,7 +369,7 @@ def _uncertainty(result):
                         self.update_results(None)
                         return
                     # When flux is equivalent to Jy, lineflux result should be shown in W/m2
-                    if flux_unit.is_equivalent(u.Unit('W/(m2 m)'/solid_angle_in_flux_unit)):
+                    if flux_unit.is_equivalent(u.W / (u.m * u.m * u.m * solid_angle_in_flux_unit)):
```

Comment:
just curious, why this swap here? there are many cases of creating units from strings that aren't changed (or possible to change, i think). is this preferred when possible?

Reply:
I didn't comb through the whole package but I change it when I see it in the diff. There is no reason to incur extra overhead of string parsing if you already know what you want.